cookie.js

```diff
@@ -0,0 +1,204 @@
+/*!
+ * cookie
+ * Copyright(c) 2012-2014 Roman Shtylman
+ * Copyright(c) 2015 Douglas Christopher Wilson
+ * MIT Licensed
```

Comment:
Please include the full LICENSE text of that file here.

Reply:
added License text and reference to the original project